scene/gui/tab_bar.cpp

```diff
@@ -323,6 +323,24 @@ void TabBar::gui_input(const Ref<InputEvent> &p_event) {
 	}
 }
 
+String TabBar::get_tooltip(const Point2 &p_pos) const {
+	for (int i = 0; i < tabs.size(); i++) {
+		if (tabs[i].hidden) {
+			continue;
+		}
+
+		if (get_tab_rect(i).has_point(p_pos)) {
+			if (tabs[i].tooltip.is_empty()) {
+				return tabs[i].text;
```

Comment:
The fact that the tab title is used if it the tooltip is empty should probably be mentioned in the docs. 
Also, the only way to not have a tooltip is to set it to whitespace.